packages/manager/apps/telecom/src/app/telecom/pack/migration/translations/Messages_fr_FR.json

```diff
@@ -105,7 +105,8 @@
   "telecom_pack_migration_confirm_no_commitment": "Ce changement d’offre n’est sujet à aucun réengagement.",
   "telecom_pack_migration_confirm_commitment": "Un réengagement de {{ month }} mois débutera à la date de facturation.",
   "telecom_pack_migration_confirm_new_offer_price": "Votre nouvel abonnement mensuel s'élève à : <span class=\"text-price\">{{ price }}</span> <span class=\"text-infos\">HT/mois</span>, applicable dès livraison de votre nouvel accès. Vous en serez informé par e-mail.",
-  "telecom_pack_migration_confirm_shipping": "Vous avez sélectionné le mode de livraison de votre nouveau modem par transporteur (+ <span class=\"text-price\">{{ transportPrice }} €</span> <span class=\"text-infos\">HT</span>). Votre première mensualité sera donc de <span class=\"text-price\">{{ price }}</span> <span class=\"text-infos\">HT</span>.",
+  "telecom_pack_migration_confirm_shipping": "Vous avez sélectionné le mode de livraison de votre nouveau modem par transporteur (+ <span class=\"text-price\">{{ transportPrice }} €</span> <span class=\"text-infos\">HT</span>).",
```

Comment:
**nitpick:** Avoid HTML in translation files. You can pass entire `<span class="text-price"> {{ transportPrice }} </span>` as the value for `transportPrice`

Reply:
ok done